featuretools/primitives/standard/transform_primitive.py

```diff
@@ -336,13 +336,20 @@ def get_function(self):
 
 
 class Haversine(TransformPrimitive):
-    """Calculate the approximate haversine distance in miles between two LatLong variable types.
+    """Calculate the approximate haversine distance between two LatLong
+        variable types. Defaults to computing in miles.
+
     """
     name = 'haversine'
     input_types = [LatLong, LatLong]
     return_type = Numeric
     commutative = True
 
+    def __init__(self, unit='miles'):
+        if unit not in ['miles', 'kilometers']:
+            raise ValueError("Invalid unit given")
```

Comment:
missing test for this scenario

Reply:
Fixed.

Reply:
let's change the text to `"Invalid unit %s provided. Must be one of ['miles', 'kilometers']" % unit`

Reply:
Fixed.